### DISCUSSION TITLE
Cropping large images for training

### DISCUSSION THREAD

**Discussion by ryanfont80** [ultralytics/yolov5#4437] [Q&A]
If we have a very large image say 10000x10000 pixels and we randomly crop it at 640x640 pixels... are those individual croppings considered unique images for training purposes? Meaning would each small image count as a "new image" for the model to study and train on?

This may be a novice question but if we made say 600 small images (even if some overlapped) out of the large image by using random cropping, it would dramatically reduce data collection for us.

Thanks!
Ryan

**Comment by glenn-jocher** [comment]
@ryanfont80 yes, but do not split a single image or video between train and val sets.

![Screenshot 2021-08-04 at 13 15 05](https://user-images.githubusercontent.com/26833433/129934554-3fd2051f-c1d7-4314-8807-d19c2941bef8.jpg)
